Assets/MixedRealityToolkit/_Core/Utilities/AsyncUtility/AwaitYieldInstructions/WaitForBackgroundThread.cs

```diff
@@ -0,0 +1,35 @@
+// MIT License
```

Comment:
I thought we agreed to not use other License texts throughout the project.
@davidkline-ms ?

Reply:
I think adding the license headers makes it very clear, in case the files get moved in the future.